documentation/bin/createDocu.sh

```diff
@@ -17,6 +17,14 @@ do
     [ -e "${f}" ] && rm -rf "${f}"
 done
 
+export GROOVY_VERSION=2.4.16
```

Comment:
I don't know how Travis CI provides the groovy runtime for the pipeline. I agree in general that this should be done on that side rather than here. Since I have no access to that side I tried to find a way do fix the compile issue by installing a new groovy.